src/test/java/seedu/kitchenhelper/storage/StorageTest.java

```diff
@@ -36,10 +38,26 @@ void getIngredientDataTest() {
 
     @Test
     void getRecipeData() {
+        try {
```

Comment:
Good job in adding test cases for your storage class! Perhaps you can add more tests to cater for your method and line coverage to cover more parts of your code for testing.